x-pack/test/functional/es_archives/spaces/copy_saved_objects/data.json

```diff
@@ -126,7 +126,7 @@
         "title": "Dashboard Foo",
         "hits": 0,
         "description": "",
-        "panelsJSON": "[{}]",
+        "panelsJSON": "[]",
```

Comment:
`{}` is definitely invalid panel shape :D 
New dashboard migration failed because of this.
Previously this would have failed on dashboard app open. 

Alternatively we could make sure that migration can swallow `{}`, but I am not sure what is a recommended approach?

Reply:
Similarly there is single change in `x-pack/test/functional/es_archives/reporting/hugedata/data.json.gz` which started failing during import. There was a typo in a reference name: `Fanel_1` reference instead of `panel_1` which led to test failures. 

I just updated the data as I guess we should fail the migration in such case.